### DISCUSSION TITLE
New Source: Criteo Ads

### DISCUSSION THREAD

**Discussion by mcfuhrt** [airbytehq/airbyte#33144] [New Connector Request]
## Tell us about the new connector you’d like to have
Criteo is an advertising company that provides online display advertisements. Criteo's product is a form of display advertising, which displays interactive banner advertisements, generated based on the online browsing preferences and behaviour for each customer. The solution operates on a pay per click/cost per click (CPC) basis.

## Are you willing to submit a PR?
Yes, we are currently working on such a connector

**Comment by igrankova** [comment]
**Webhook-based? (no/partially/yes)** 
no

**Available authentication modes (API key/Oauth/other)** 
OAuth:
https://developers.criteo.com/marketing-solutions/reference/oauth

**Has Sandbox environment?** 
No
Developer environment
https://partners.criteo.com/

**How to populate the account with data** 
API:
https://developers.criteo.com/marketing-solutions/reference/oauth
https://support.criteo.com/s/article?article=360001226005-Marketing-API-details&language=en_US

**Available streams for sync** 
advertisers
audiences
campaigns
categories

**Integration supports incremental sync?** 
No

**Other information/blockers** 
API Error Types:
https://developers.criteo.com/marketing-solutions/docs/how-to-handle-api-errors

**Comment by igrankova** [comment]
@sherifnada Vetting is completed, ready for implementation

**Comment by Timopheym** [comment]
Hey dear @mcfuhrt would you be please able to update on your status on a connector? We are also have a need for it, and probably would manage the situation with the custom script but it would be much preferable to have it and a part of our Airbyte configuration!
Thanks!

**Comment by transistormuncher** [comment]
Hi everyone, we would also be interested in this connector. What is needed to get it implemented in the next airbyte version?
Thanks for the great work!

**Comment by indyMccarthy** [comment]
Hello ! Any news on this issue ?
Thanks !

**Comment by igrankova** [comment]
@YowanR, we have some requests to this new connector here, is it in our roadmap?

**Comment by MRKCart** [comment]
We would be extremely interested in this connector.

**Comment by mcfuhrt** [comment]
We have implemented the connector at a customer, I will look it up again and adapt it if necessary :)

Sorry for the late feedback!

**Comment by transistormuncher** [comment]
Thank you @mcfuhrt, this would still be much appreciated!

**Comment by YowanR** [comment]
@mcfuhrt any updates here, please?

**Comment by carlonuccio** [comment]
I have implemented the connector, the ads report. I opened a PR https://github.com/airbytehq/airbyte/pull/25241
